crash-reporting-core/src/commonMain/kotlin/dev/icerock/moko/crashreporting/core/CrashReportingCore.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.crashreporting.core
+
+class CrashReportingCore {
+
+    fun getLogText(logLevel: String? = null, tag: String? = null, message: String? = null): String {
+        var logText = logLevel?.let { "$it:" }.orEmpty()
+
+        tag?.let { logText = logText.plus(" $tag - ") }
+        message?.let { logText = logText.plus(message) }
+        return logText
+    }
+
+    fun getExceptionName(throwable: Throwable): String? {
+        return throwable.noReflectionSimpleName
```

Comment:
simpleName already available in common 
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect/-k-class/simple-name.html
maybe it will work on 1.4.10 on android without reflection now?